configuration/wireless/headless.md

```diff
@@ -15,12 +15,12 @@ update_config=1
 country=<Insert 2 letter ISO 3166-1 country code here>
 
 network={
- ssid="<Name of your WiFi>"
- psk="<Password for your WiFi>"
+ ssid="<Name of your wireless LAN>"
+ psk="<Password for your wireless lAN>"
```

Comment:
lAN -> LAN

Reply:
That particular lAN was actually called Ian. Wish I could type, then re-read what I have typed and actually see the mitakes.